documentation/en/join-devnet-cli.md

```diff
@@ -1,12 +1,12 @@
-# Join Lotus Devnet
+# Join DevNet
```

Comment:
`Devnet` is one word

Reply:
Other companies abbreviate Devnet as `DevNet`, an example is Cisco.